core/unix/rseq_linux.h

```diff
@@ -58,7 +58,7 @@ bool
 rseq_is_registered_for_current_thread(void);
 
 void
-rseq_locate_rseq_regions(void);
+rseq_locate_rseq_regions(bool);
```

Comment:
Give the param a name -- but see above where I think this param will go away

Reply:
Gave it a name.